Tests/Rules/PlaceCloseBrace.tests.ps1

```diff
@@ -80,19 +80,31 @@ $hashtable = @{a = 1; b = 2}
 
     Context "When there is a multi-line hashtable" {
         BeforeAll {
-            $def = @'
-$hashtable = @{
-    a = 1
-    b = 2}
-'@
+
             $ruleConfiguration.'NoEmptyLineBefore' = $false
             $ruleConfiguration.'IgnoreOneLineBlock' = $true
             $ruleConfiguration.'NewLineAfter' = $false
-            $violations = Invoke-ScriptAnalyzer -ScriptDefinition $def -Settings $settings
         }
 
         It "Should find a violation" {
+            $def = @'
+            $hashtable = @{
+                a = 1
+                b = 2}
+'@
+            $violations = Invoke-ScriptAnalyzer -ScriptDefinition $def -Settings $settings
+            $violations.Count | Should -Be 1
+        }
+
+        It "Should not crash when hashtable is defined on first token" {
+            $def = @'
+            @{
+                key = value }
+'@
+
+            $violations = Invoke-ScriptAnalyzer -ScriptDefinition $def -Settings $settings -ErrorAction Stop
             $violations.Count | Should -Be 1
+            Invoke-Formatter -ScriptDefinition $def -ErrorAction Stop
```

Comment:
There doesn't seem to be an assertion here. Since the formatter uses the same rule, this seems superfluous. (do you want an explicit test for the formatter?)

Reply:
The assertion here is that the formatter does not throw an exception. The call to `Invoke-ScriptAnalyzer` is the core unit test but the call to `Invoke-Formatter` is the integration test.